recipes/pyscenic/build.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e  
+patch -p1 < ${RECIPE_DIR}/dill_patch.patch
```

Shellcheck

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)